envoy/conf.yaml.example

```diff
@@ -0,0 +1,38 @@
+# This file is overwritten upon Agent upgrade.
+# To make modifications to the check configuration, please copy this file
+# to `envoy.yaml` and make your changes on that file.
+
+init_config:
+
+instances:
+  # For every instance, you need a `stats_url` and can optionally
+  # supply a list of tags. The admin endpoint must be accessible.
+  # https://www.envoyproxy.io/docs/envoy/latest/operations/admin
+
+  - stats_url: http://localhost:80/stats
+
+  #   tags:
+  #     - instance:foo
```

Comment:
In case you missed, probably you can use fixed string tags to distinguish Envoy instances. https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/metrics/v2/stats.proto#config-metrics-v2-tagspecifier I don't know the tag can work with admin `/stats`, but I'm using the fixed tag with dog_statsd sink. Please note the fixed tag feature is only available with Envoy >= v1.6.0.

Reply:
Thanks! We actually allow users to specify any tag they wish, which is especially useful for distinguishing between different instances of an integration for metric aggregation.